app/src/main/java/com/github/onlynotesswent/ui/overview/FolderContent.kt

```diff
@@ -138,6 +142,11 @@ fun FolderContentScreen(
               context = context,
               folder = folder.value)
         }) { paddingValues ->
+          Button(onClick = { showFileSystemPopup = true }) { Text("FileSystem Overview") }
```

Comment:
Consider adding the Text string to the strings.xml file

Reply:
Yes good point